### ISSUE TITLE
branch 2.0 is broken after adding support for returning JS values

### ISSUE THREAD

**Issue by manolo** [vaadin/flow#5325]
This PR broke 2.0 https://github.com/vaadin/flow/pull/5285

- Reproduce
Checkout beverage branch 2.0, run `mvn jetty:run` navigate to the app and you will get errors of the kind 

```
Caused by: java.lang.NoSuchMethodError: com.vaadin.flow.dom.Element.callFunction(Ljava/lang/String;[Ljava/io/Serializable;)V
    at com.vaadin.flow.component.combobox.ComboBox.setDataProvider(ComboBox.java:488)
    at com.vaadin.flow.component.combobox.ComboBox.setDataProvider(ComboBox.java:603)
    at com.vaadin.flow.component.combobox.ComboBox.setDataProvider(ComboBox.java:556)
    at com.vaadin.flow.component.combobox.ComboBox.setItems(ComboBox.java:390)
    at com.vaadin.starter.beveragebuddy.ui.views.reviewslist.ReviewEditorDialog.createCategoryBox(ReviewEditorDialog.java:98)
    at com.vaadin.starter.beveragebuddy.ui.views.reviewslist.ReviewEditorDialog.<init>(ReviewEditorDialog.java:54)
    at com.vaadin.starter.beveragebuddy.ui.views.reviewslist.ReviewsList.<init>(ReviewsList.java:75)
```

- Workaround
revert 16b81987eec54f72aa21accc4b84d9600608eab3

**Comment by caalador** [comment]
This is a no issue. The real fix is to build ComboBox against Flow 2.0 as now there is such changes that the jar built against 1.5-SNAPSHOT is not valid anymore against 2.0-SNAPSHOT

**Comment by caalador** [comment]
This goes for all components that use the `callFunction` api.
